codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/HyperFuelWorker.res

```diff
@@ -33,41 +32,41 @@ let makeGetRecieptsSelectionOrThrow = (~contracts: array<contract>) => {
       txStatus: txStatusSelection,
     }: HyperFuelClient.QueryTypes.receiptSelection
   )
-  let maybeWildcardLogSelection = switch wildcardRbs {
-  | [] => None
-  | wildcardRbs =>
-    Some(
-      (
-        {
-          receiptType: logDataReceiptTypeSelection,
-          txStatus: txStatusSelection,
-          rb: wildcardRbs,
-        }: HyperFuelClient.QueryTypes.receiptSelection
-      ),
-    )
-  }
 
   contracts->Array.forEach(contract => {
     let nonWildcardRbs = []
     contract.events->Array.forEach(event => {
-      let module(Event) = event.eventMod
-      let {isWildcard} = Event.handlerRegister->Types.HandlerTypes.Register.getEventOptions
       switch event {
-      | {mint: true} => mint := (isWildcard ? #Wildcard : #NonWildcard)
+      | {mint: true, isWildcard: true} => mint := #Wildcard
+      | {mint: true} => mint := #NonWildcard
       | {logId} => {
           let rb = logId->BigInt.fromStringUnsafe
-          if isWildcard {
+          if event.isWildcard === Some(true) {
```

Comment:
Maybe this should go in the pattern match like the above cases?

Reply:
I didn't want to duplicate the `let rb = logId->BigInt.fromStringUnsafe` line